Extensions/xkit_patches.js

```diff
@@ -388,8 +388,9 @@ XKit.tools.getParameterByName = function(name){
 
 			get_tumblelog: function() {
 				var new_channel_id = document.location.href.match(/[&?]tumblelogName=([\w-]+)/);
+				var archive_channel_id = document.location.href.match(/[&?]name=([\w-]+)/);
```

Comment:
The iframe controls used on blogname.tumblr.com/archive has a different URL structure than the one used on blogname.tumblr.com.